### ISSUE TITLE
Cannot handle alerts deleted in Wavefront GUI

### ISSUE THREAD

**Issue by stephenchu** [spaceapegames/terraform-provider-wavefront#32]
Version: 1.0.2, I noticed a change in behavior from 1.0.1. Also Terraform v0.10.7 (but shouldn't matter).

With 1.0.1, if I have a Terraform managed (and applied) alert `wavefront_alert.foo`, if I deleted this WF alert manually using WF's web gui, the next time when I use Terraform to plan this alert, since my `.tf` file still contains its definition, Terraform plan tells me that it wants to create this alert as new:

`+ wavefront_alert.foo`

My course of action in this case is, to remove such alert definition from the `.tf` file.

With 1.0.2, the same workflow, but I can't even plan successfully, but with the following error:

```
wavefront_alert.foo: Refreshing state... (ID: 1234567890)
Error refreshing state: 1 error(s) occurred:

* wavefront_alert.1234567890: wavefront_alert.1234567890: Error finding Wavefront Alert . server returned 404 Not Found
{"status":{"result":"ERROR","message":"Alert 1234567890 does not exist","code":404}}
```

... and even after I remove its `.tf` definition, my `terraform plan` still errors out. At this point, I cannot `terraform plan -destroy -target=wavefront_alert.foo` (will fail with 404). I now have to resort to `terraform state rm` which is extremely unpleasant.

I think PR #27 is causing this behavior change to happen. I also think the former 1.0.1 behavior is more Terraform-esque from all the providers I've worked with. Do you agree?

**Comment by nathmclean** [comment]
Yeah the change from using Find to Get seems to be the problems (find returns an empty list, whereas Get errors when a resource doesn't exist)

I've just tested a fix. I'll need some more time to make sure all the tests still pass and to check other resources. Hopefully I can get this out by the end of the week.

**Comment by nathmclean** [comment]
Fixed by #33 (Check for 404s returned when a resource has been deleted out of band)
